_candidates_ja/KonradMalawski_1.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Konrad `ktoso` Malawski
+title: "あらゆるものを分散コンピューティングで扱おう(Akkaをつかって)"
+length: 40
+audience: Intermediate
+language: English
+twitter: ktosopl
+github: ktoso
+icon: https://scontent-nrt1-1.xx.fbcdn.net/v/t31.0-8/15626425_10202875601369012_2764589549729738273_o.jpg?oh=57591212f91d0b834b45ff9806e05ec8&oe=5A921F16
+organization: Akka Team, Lightbend
+tags:
+  - Software Design and Architecture
+  - Microservices
+  - Others
+suggestions:
+  - 高性能・高信頼性のシステムに興味のある開発者、このセッションを聴講するのにAkkaに関する事前知識は必要はありません
```

Comment:
Developers interested in building performant and reliable systems. No prior exposure to Akka is needed to enjoy the session.